etc/grafana/dashboards/reth-mempool.json

```diff
@@ -2425,31 +2425,117 @@
           },
           "disableTextWrap": false,
           "editorMode": "builder",
-          "expr": "rate(reth_network_transaction_fetcher_eip2930_sum{instance=\"$instance\"}[$__rate_interval])",
+          "expr": "reth_network_duration_fill_request_from_hashes_pending_fetch{instance=\"$instance\"}",
           "fullMetaSearch": false,
           "hide": false,
-          "includeNullMetadata": false,
+          "includeNullMetadata": true,
           "instant": false,
-          "legendFormat": "Eip2930",
+          "legendFormat": "Fill Request",
           "range": true,
           "refId": "B",
           "useBackend": false
+        }
+      ],
+      "title": "Fetch Hashes Pending Fetch Duration",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "Durations of one call to poll `NetworkManager` future, and its nested function calls.\n\nNetwork Handle Message - stream network handle messages from `TransactionsManager`;\nSwarm Events - stream transaction gossip from `Swarm`",
```

Comment:
since this is user facing i'd err on not naming structs etc, but not a blocker

Reply:
yeah in general same...don't think these types will disappear though, hence went for it. also see major benefits of lowering barriers to entry for contributors looking at public dashboard, hence don't have the long metric names to search the codebase to match the panel with the code.